tests/operators/_asynchronous/test_bigquery.py

```diff
@@ -54,7 +54,7 @@ def test_dbt_run_airflow_async_bigquery_operator_base_cmd(profile_config_mock):
 @patch.object(DbtRunAirflowAsyncBigqueryOperator, "build_and_run_cmd")
 def test_dbt_run_airflow_async_bigquery_operator_execute(mock_build_and_run_cmd, profile_config_mock, monkeypatch):
     """Test execute calls build_and_run_cmd with correct parameters."""
-    monkeypatch.setattr("cosmos.operators._asynchronous.bigquery.enable_setup_async_task", False)
+    monkeypatch.setattr("cosmos.operators._asynchronous.bigquery.settings.enable_setup_async_task", False)
```

Comment:
Why did we use `monkeypatch.setattr` instead of the `patch` decorator, that is most commonly used in Cosmos?
```
@patch("cosmos.operators._asynchronous.bigquery.settings.enable_setup_async_task", return_value=False)
```

Reply:
The initial idea was to set this value at runtime, but since I have changed the import path, the `patch` should also work. Pushed a fix in d4f545836f83fefcf193fc7ce0ff92f4c0d82b27